src/module/data/actor/character.mjs

```diff
@@ -136,6 +136,21 @@ export default class CharacterModel extends BaseActorModel {
     this.potency.strong += highestCharacteristic;
   }
 
+  /** @override */
+  async _preCreate(data, options, user) {
+    await super._preCreate(data, options, user);
```

Comment:
Although we don't currently ever return false in BaseActorModel, it's still good practice to include
```suggestion
    const allowed = await super._preCreate(data, options, user);
    if (allowed === false) return false;
```

Reply:
Fair enough. I'll keep that in mind for the future.